### DISCUSSION TITLE
5.17 对本科生论文模版进行了更新

### DISCUSSION THREAD

**Discussion by peterzheng98** [sjtug/SJTUThesis#620] [灌水区]
有无更新之后的tex？

**Comment by JunyuanDeng** [comment]
同求

**Comment by skyzh** [comment]
请提供一个样例 pdf（

**Reply by JunyuanDeng** [reply]
[中文.pdf](https://github.com/sjtug/SJTUThesis/files/6493818/20210517085840680871.pdf)
[英文.pdf](https://github.com/sjtug/SJTUThesis/files/6493820/20210517085741636493.pdf)
